.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_API_ENDPOINT=https://api.futurelabourmps.com/
-REACT_APP_API_COLS=A3:AZ70
\ No newline at end of file
+REACT_APP_API_COLS=A3:AZ70
+POSTHOG_API_KEY=phc_tRkyMnTTjz6EXTGDxTBw4FgZacvm3MdsXd7aK485Gnh
```

Comment:
Is this sensitive? It might be better in the `.env.local` file if so.

Reply:
It's a public key :) Ultimately it's running on the client so you can't really hide it!